pandas/core/frame.py

```diff
@@ -4207,17 +4207,23 @@ def update(self, other, join='left', overwrite=True, filter_func=None,
                raise_conflict=False):
         """
         Modify DataFrame in place using non-NA values from passed
```

Comment:
May have to reword this so the first line isn't too long? Does the validation script complain about this?

Reply:
The validation script actually did not complain, but I'll try to adjust it.